extensions/src/main/java/org/pentaho/platform/plugin/action/ActionRunner.java

```diff
@@ -68,14 +70,17 @@ public ActionRunner( final IAction actionBean, final String actionUser, final Ma
   }
 
   public Boolean call() throws ActionInvocationException {
+    final String workItemUid = ActionUtil.extractUid( params );
     try {
-      return callImpl();
+      final boolean result = callImpl();
+      WorkItemLifecyclePublisher.publish( workItemUid, params, WorkItemLifecyclePhase.SUCCEEDED );
```

Comment:
One of the things that was mentioned when @pentaho-mzur and @chiltonj shared the LifecyclePhase events with the Rogue One team was that it could make sense to reword them, so that its nomenclature would match the nomenclature seen on Mesos UI. This would help a customer to more easily to viasually tie in the lifecycle phases, regardless of looking at Mesos UI , logs or Monitoring traps.

Mesos UI has FINISHED. WorkItemLifecyclePhase has SUCCEEDED. 

Is this rewording going to happen? Or has this been deemed not necessary? 
Roping in on @pentaho-mzur / @chiltonj to help me recall this.